compute_tools/src/compute.rs

```diff
@@ -1459,12 +1459,18 @@ impl ComputeNode {
         let tls_config = self.tls_config(&pspec.spec);
         let databricks_settings = spec.databricks_settings.as_ref();
 
+        let postgres_port = self
+            .params
+            .connstr
+            .port()
+            .expect("port must be present in connstr");
```

Comment:
It looks like all e2e tests fail because of that. I think because cplane doesn't pass port in the connection string, so it's undefined here
https://github.com/neondatabase/cloud/blob/07c134f2de939a6497f360098504984a9b1529f2/goapp/controlplane/internal/pkg/compute/provisioner/provisioner_common.go#L68

You probably need to specify the default somewhere in the compute_ctl